4_visualize.R

```diff
@@ -0,0 +1,44 @@
+source('4_visualize/src/plot_results.R')
+
+p4 <- list(
+  # Plots for each lake-gcm combo
```

Comment:
I am a bit nervous about having these first three plotting targets run for all lakes (unless I am misunderstanding what they would do). This would create an enormous amount of plots. By my count `p4_20yr_gcm_preds_ice` would produce 21,498 plots (3,583 lakes x 6 GCMs), `p4_20yr_average_preds_ice` would produce 3,583 plots (one for each lake), and `p4_20yr_average_profiles` would also produce 3,583 plots (one for each lake). That's a total of over 28k plots!

So, if someone just ran this pipeline that would eat up A TON of time and memory and would probably wouldn't even really look at many. It would be too difficult to find the one we were looking for. 

I think these plots are useful for debugging, though. Especially, when we have some site-specific questions. So I understand the need to have a way to make site-specific plots. I wonder if we can handle this by adding a target that is a vector of site ids (for now, could do a random sample), that could be changed to rebuild these plots and subset the mapping variable tables that go into building them. Thoughts?

Reply:
yup, agreed -- we definitely don't want to be generating these for every lake, and that was never the intent. I'll make that change now